lib/core/macros.nim

```diff
@@ -980,9 +1020,10 @@ macro dumpAstGen*(s: untyped): untyped = echo s.astGenRepr
   ## Outputs:
   ##
   ## .. code-block:: nim
-  ##    nnkStmtList.newTree(
+  ##
+  ##    newStmtList(
   ##      nnkCommand.newTree(
```

Comment:
'.newTree'

Reply:
No, `newStmtList` is correct here. As I said, "I instructed the generator to use some shorter notations when possible". The `newStmtList` constructor proc is just one example.

Reply:
Sorry, I was referring to the last line `nnkCommand.newTree(`